telegram/ext/filters.py

```diff
@@ -213,7 +215,7 @@ def filter(self, message):
             return True
 
     all = _All()
-    """:obj:`Filter`: All Messages."""
```

Comment:
The rendered page is a lot shoter to scroll through for me. It just felt like visual clutter. Also kinda confusing to be honest since there is no "Filter" object.

Reply:
seems like differences in sphinx versions. Keep it like in this PR is fine